src/ExchangeMapping.ts

```diff
@@ -136,6 +137,16 @@ export function handleFill(event: OrderFilled): void {
 
   updateTradesQuantity(orderBook, side, orderId);
 
+  // Update market position
+  updateMarketPositionFromOrderFilled(
```

Comment:
where is taker updated? or are they updated as a maker (from their own distinct OrderFilled event)?

Reply:
> or are they updated as a maker (from their own distinct OrderFilled event)

Yes exactly, OrderFilled is emitted on each individual order fill so it is updated as the maker on its own OrderFilled event